app/data/jsonSchemaForm/projectManagerSchema.ts

```diff
@@ -2,7 +2,6 @@ const projectManagerSchema = {
   $schema: "http://json-schema.org/draft-07/schema",
   type: "object",
   title: "Project Manager",
-  required: ["cifUserId"],
```

Comment:
This is an interesting one, as this property is actually required in the database. Can we leave a comment below to explain why we don't want it to be required in the JSON schema?